app/code/Bogdank/SupportChat/view/frontend/web/js/view/chat-form.js

```diff
@@ -0,0 +1,66 @@
+define([
+    'jquery',
+    'ko',
+    'uiComponent',
+    'Magento_Customer/js/customer-data',
+    'Bogdank_SupportChat/js/action/send-message',
+    'Magento_Ui/js/modal/modal'
+], function ($, ko, Component, customerData, sendMessage) {
+    'use strict';
+
+    return Component.extend({
+        defaults: {
+            customerMessage: customerData.get('customer-message'),
+            messages: []
+        },
+
+        message: ko.observable(),
+        customername: ko.observable(),
+
+        /** @inheritDoc */
+        initialize: function () {
+            this._super();
+
+            $(document).on(
+                'bogdank_SupportChat_openModal.bogdank_supportChat',
+                $.proxy(this.openModal, this)
+            );
+
+            this.messages.forEach(function (messageData) {
+                messageData.value = ko.observable('');
+            });
+
+        },
+
+
+        /**
+         * Init modal from the component HTML
+         */
+        initModal: function (formSection) {
+            this.modal = $(formSection).modal({
+                buttons: []
+            });
+        },
+
+        /**
+         * Open modal form with chat for message
+         */
+        openModal: function () {
+            this.modal.modal('openModal');
+        },
+
+        /**
+         * Send message
+         */
+        sendMessage: function () {
```

Comment:
@holmatro90 Validation call missed. The form can be submitted without the fields validation.

P.S.: Pressing `X` button near the chat button generates error because this functionality is not relevant anymore.